<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modifierconteneur</class>
 <widget class="QDialog" name="modifierconteneur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>156</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="TypeConteneur">
     <property name="text">
      <string>TypeConteneur :</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="CapaciteMax">
     <property name="text">
      <string>CapaciteMax :</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="IdConteneur">
     <property name="text">
      <string>IdConteneur :</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="Annuler">
     <property name="text">
      <string>Annuler</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="Modifier">
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="IdConteneur_2"/>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLabel" name="Titre">
     <property name="font">
      <font>
       <family>Sitka</family>
       <pointsize>13</pointsize>
       <italic>false</italic>
       <bold>false</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Modifier Conteneur :</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QComboBox" name="TypeConteneur_2"/>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QDoubleSpinBox" name="CapaciteMax_2"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
